### PR TITLE
Joy's Kuda bank app user guide

### PR DESCRIPTION
**Pull request by JoyNwaiwu**
Please take a look at my pull request. @owans

### CODE REVIEW

**Comment by JoyNwaiwu** [comment]
Hi @Roxie-32, please take a look.

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    2.3   If you were referred by someone and have a referral code, you can type the code. Else, skip that and click Next
+    2.4   Now, you’ll have to confirm the email address you imputed previously. Get the Verification Code from your email and enter it. Click Next
+    
+    N/B: If, for some reason, you checked your email and didn’t see the code, you can select Resend code.
+
+![](https://i.imgur.com/Nn0p86m.png)
+
+**Congratulations! You have successfully signed up and opened an account.**
+    
+#### 3. Setting up your profile <a name="settingupyourprofile"></a>
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:62]: I don't see images for these steps

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+1. [Download the app on Google Play Store and launch it](#downloadtheappongoogleplaystoreandlaunchit)
+2. [Open the app and follow the prompts on your screen to open an account](#opentheappandfollowthepromptsonyourscreentoopenanaccount)
+3. [Setting up your profile](#settingupyourprofile)
+4. [Sign in to your account](#signintoyouraccount)
+5. [Verify your identity](#verifyyouridentity)
+6. [Secure your account with a 4-digit pin to confirm transactions](#secureyouraccountwitha4-digitpintoconfirmtransactions)
+7. [Enter more details about yourself and how you’ll use the app](#entermoredetailsaboutyourselfandhowyou’llusetheapp)
+8. [Add a valid ID to increase your transaction limit](#addavalididtoincreaseyourtransactionlimit)
+
+[Kuda savings](#kudasavings)
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:19]: No numbering on this, kuda card, fees and conclusion

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+
+* Be a minimum of 16 years old and a Nigerian citizen
+* A smartphone that supports the latest version of the KUDA app (for this guide, we’re using Android)
+* An active phone number and email address
+
+## How to signup <a name="howtosignup"></a>
+
+To sign up quickly and create an account on the Kuda Bank app, follow these steps below:
+
+#### 1. Download the app on Google Play Store and launch it. <a name="downloadtheappongoogleplaystoreandlaunchit"></a>
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:46]: Add images for these

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+        -   LGA
+        -   City
+        -   House number
+        -   Street name
+        
+At this point, you have successfully created your account! You should see a “**Welcome to Kuda message**” on your screen. Click Continue!
+
+![](https://i.imgur.com/DogEbzw.png)
+
+#### 4. Sign in to your account <a name="signintoyouraccount"></a>
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:84]: Show yourself actively doing these so the user doesn't feel bored or along when going through this guide

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+* Confirm your phone number
+* Type in the name you want us to print on your card
+* Confirm your delivery details
+
+## Kuda fees <a name="kudafees"></a>
+Kuda gives 25 free transfers every month if you sign up with your Bank Verification Number (BVN). Without your BVN, you will get 2 free transfers every month. Extra transfers will cost you 10 naira each.
+
+Everything else is free, including using your account, using your Kuda card, and withdrawing cash from Kuda’s partner ATMs.
+
+## Conclusion <a name="conclusion"></a>
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:136]: Always summarize what has been done or achieved

File: TMP2022/technical-writing-track/joy-tw-owans/README.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+* [Fixed savings](#fixedsavings)
+
+[Kuda card](#kudacard)
+[Kuda fees](#kudafees)
+[Conclusion](#conclusion)
+
+
+## Overview <a name="overview"></a>
+
+Among the latest trending bank apps in Nigeria presently is the Kuda Bank app. Created as an online-only bank without any physical branch has made it stand out to most people, especially the Nigerian youths. The Kuda Bank is one of the most well-designed fintech companies in Nigeria, packed with unique features that make it an excellent replacement for the conventional Nigerian banking system.
```
Comment (owans) [inline TMP2022/technical-writing-track/joy-tw-owans/README.md:31]: Add a summary of the steps the user should expect in the introduction

**Review by S-code652 — APPROVED** [review]

**Pull request merged**